docs/whats-new.md

```diff
@@ -1,53 +1,36 @@
-
-A new `ViewportLayout` React component makes it trivial to precisely position multiple base maps and other background HTML components under multi-viewport layouts, automatically taking care of complexities like `devicePixelRatio`, inverted WebGL/CSS coordinate systems etc. This component will also automatically hide base maps when the viewport parameters can not be supported by the underlying map component.
-
-
-## All Viewports Now Geospatially Enabled
-
-All `Viewport` classes are now geospatially enabled: they now take an optional `longitude`/`latitude` reference point. In this mode, `position`s will be treated as meter offsets from that reference point per `COORDINATE_SYSTEM.METER_OFFSET` conventions.
-
-This means that you can now use a FirstPersonViewport (the successor to the `PerspectiveViewport` with layers encoded in `COORDINATE_SYSTEM.LNG_LAT` and `COORDINATE_SYSTEM.METER_OFFSETS`) and place a camera anywhere in the scene (in contrast to the `WebMercatorViewport` which only allows you to look "down" on a position on the map).
-
-Viewports even accept a `modelMatrix` to allow viewport/camera positions to be specified in exactly the same coordinates as `METER_OFFSET` layers, making it possible to place a camera at the exact location any of your existing data points without having to think or do any math.
+Release date: TBD, Dec 2017
 
+All new additions to the official deck.gl 4.2 API are listed here. Note that in addition to the official new features in this release, deck.gl 4.2 also contains a number of signficant under the hoods changes to prepare for new features and optimizations. Some of these are available as experimental APIs, see below.
 
-## WebVR Support and Example
+As always, for information on deprecations and how to update your code in response to any API changes, please read the deck.gl [Upgrade Guide](/docs/get-started/upgrade-guide.md).
 
-Multi viewport support can be used to integrate with the WebVR API and create dual WebVR compatible viewports that render a first person view of your data for left and right eye respectively which will display as stereoscopic 3D in supporting hardware.
+> TODO - Add a row of small square GIFs/PNGs of Viewport Transitions, Automatic Highlighting, Dashed Lines, Multiple Viewports. Would be really nice with a screenshot (or GIF) of Seer.
 
 
-## Controller Classes
+## DeckGL: Viewport Transitions
 
-deck.gl's controller classes have been significantly refactored, providing you with lots of options for how to control how the user interacts with your viewports. Also the controllers are no longer categorized as "experimental" exports, so you can import and use them in your apps knowing that they will be supported in future releases the same way as any other deck.gl feature.
+A major new feature is support for Viewport Transitions.
 
-TBA...
+> TODO - Add more extensive description. This is our major official feature after all...
```

Comment:
I will add this section once viewportController/DeckGL merging is finalized. Thanks for `TODO` note.